modules/kn-service-apply.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_serving/serverless-applications.adoc
+// * serverless/cli_reference/kn-services-ref.adoc
+
+[id="kn-service-apply_{context}"]
+= Applying service declarations
+
+You can apply declarations to a Knative service by using the `kn` CLI `kn service apply` command.
+
+.Example commands
+
+* Create a service:
++
+[source,terminal]
+----
+$ kn service apply <service-name> --image <image>
+----
+
+* Add an environment variable to a service:
++
+[source,terminal]
+----
+$ kn service apply <service-name> --image <image> --env <key>=<value>
+----
++
+[NOTE]
+====
+All parameters must be fully specified when using `kn service apply` to add an environment variable.
+====
+
+* Read the service declaration from a file:
```

Comment:
Would it make sense to add, that this file should be a json or yaml file (with the appropriate file extension) and contain the service description ?

Reply:
> contain the service description

Do you mean to provide an example output? Happy to include this if you can provide it?

Reply:
You mean the output returned by `kn` after issuing this command with a file ? I can deliver this tomorrow when I have a running cluster again.